src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -273,7 +273,7 @@ private boolean checkIf(final DetailAST ast, boolean useBreak,
             isTerminated = isTerminated(elseStmt.getFirstChild(),
                 useBreak, useContinue);
         }
-        else if (elseStmt == null) {
+        else {
             isTerminated = false;
         }
```

Comment:
TC:
https://teamcity.jetbrains.com/viewLog.html?buildId=2569271&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsPullRequest
> FallThroughCheck.java (1)
272: checkIf() If statement can be replaced with '&&'

I assume it wants me to merge all the expressions into 1 assignment.

Reply:
A sevntu check then wanted `elseStmt != null` to be before the method calls.
PMD then wanted the local variable `isTerminated` before the return removed.